tox.ini

```diff
@@ -1,30 +1,27 @@
 [tox]
-envlist = py27,py35,py37
+envlist = {py27,py35,py37}{,-gui}
 minversion = 2.0
 
 [testenv]
 deps = -r{toxinidir}/requirements.txt
-commands = py.test -v --ignore=tests/test_console.py tests/
+       gui-py27: pyside==1.2.4
+       gui-!py27: pyqt5==5.10.0
```

Comment:
What's the etymology of `!`?

Reply:
that `!py27` is not a dash-limited part of the environment name

Reply:
Again, would prefer an explicit list instead of magic `!`.